plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -953,9 +953,6 @@ protected boolean isSupportedJoinCondition(ConnectorSession session, JdbcJoinCon
     @Override
     protected String createTableSql(RemoteTableName remoteTableName, List<String> columns, ConnectorTableMetadata tableMetadata)
     {
-        if (tableMetadata.getComment().isPresent()) {
```

Comment:
Why this got removed in this commit?

Reply:
The validation was moved into the new added method `getExtraTableAndColumnComments` in this implementation